lib/src/common/abstract_open_factory.dart

```diff
@@ -21,8 +29,24 @@ class SqliteOpenOptions {
   /// Whether this connection is read-only.
   final bool readOnly;
 
+  /// Mutex to use in [SqliteConnection]s
+  final Mutex? mutex;
+
+  /// Name used in debug logs
+  final String? debugName;
+
+  final SerializedPortClient? upstreamPort;
```

Comment:
I think this can be removed - the caller of `openConnection` can use the `SqliteConnection.updates` stream directly. The web implementation likely doesn't need a SerializedPortClient at all. For native, the SerializedPortClient can probably be created when opening the connection (or isolateConnectionFactory), instead of in the SqliteDatabase.

Reply:
This has been removed from the open options. The `SqliteConnection` will now either accept an `upstreamPort` or create it's own `eventsPort` and expose updates directly.